data/json/recipes/other/vehicles.json

```
@@ -1505,5 +1505,41 @@
     "book_learn": [ [ "textbook_computer", 1 ], [ "advanced_electronics", 1 ], [ "reference_fabrication1", 1 ] ],
     "using": [ [ "3d_printing_standard", 10 ] ],
     "components": [ [ [ "carbon_fiber_filament", 100 ] ] ]
+  },
+  {
+    "result": "frame_plastic",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_VEHICLE",
+    "skill_used": "computer",
+    "difficulty": 1,
+    "autolearn": true,
+    "time": "20 m",
+    "using": [ [ "3d_printing_standard", 10 ] ],
```

Number matches the amount of plastic chunks used.
```suggestion
    "using": [ [ "3d_printing_standard", 20 ] ],
```